handlers/deployment.rb

```diff
@@ -52,12 +52,21 @@ def rebuild_subject_set_search_api(response)
       end
 
       def tag_deploy(response)
-        jenkins_job_name = JOBS.fetch('deploy')
         repo_name = repo_name_without_whitespace(response.matches[0][1])
-        track_deploy_data_in_redis(repo_name)
-        build_jenkins_job(response, jenkins_job_name, { 'REPO' => repo_name })
-      rescue Lita::Github::StatusReporter::UnknonwnRepoCommit => e
-        response.reply("Failed to deploy: #{e.message}")
+        begin
+          output = config.github_status_reporter.update_production_tag(repo_name)
```

Comment:
this object isn't a `github_status_reporter` anymore as now it's interacting with the GH API for non reporting functionality. So I think we should rename this for clarity about the class / object's responsibility

Reply:
can't argue with that